http-assert/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for http-assert 1.2
+// Project: https://github.com/jshttp/http-assert
+// Definitions by: jKey Lu <https://github.com/jkeylu>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare interface createAssert {
+    (value: any, status?: number, msg?: string, opts?: {}): void;
+    equal(a: any, b: any, status?: number, msg?: string, opts?: {}): void;
+    notEqual(a: any, b: any, status?: number, msg?: string, opts?: {}): void;
```

Comment:
`opts` isn't documented in the README there; do you know anything about what options are available?